Bugsnag process

lukehansell opened the pull request:
I wanted to document our Bugsnag process since it's something that we each do so infrequently. This should act as a guide as to what is expected of you when you work on Bugsnag, but not a definitive checklist.

Thoughts, comments, questions and amendments are all welcome

Also, this is the process as I understand it. I may have got it wrong/ missed info. Correct me, it's okay.
- [x] +1 I like it
- [x] + 2 I like it also
- [ ] + 3 I'm just following the crowd

Commit:
adding md file on bugsnag process

Commit:
adding spacing for images

Commit:
adding spacing for FAQ

Joezo commented on bugsnag-process.md:
```diff
@@ -0,0 +1,149 @@
+
+You can use the search form on the left hand side to limit what bugs you see:
+
+![](htps://cldup.com/hmbTaL9r6A-2000x2000.png =x100)t
```
t

Commit:
reomving t

SkylerAvery commented:
We need to add in a section that on the end of your bugsnag slot, to clear all old bugs that have not been appearing for more than 2 weeks or so. That way we can prevent the huge build up that myself, @rahulpatel and @andrewcashmore had when we were on bugsnag :)

lukehansell commented:
Is that a standard action now @LouisPryer? I wasn't even aware of it. Can anyone second this? If so I'll add it in.

robhuzzey commented:
+1 excellent stuff, thanks

stuartstobie commented:
certainly seems like reasonable practice to remove tickets if the snag hasn't been seen for a fortnight, I'd give that a +1

stuartstobie commented on bugsnag-process.md:
```diff
@@ -0,0 +1,149 @@
+
+The Bugsnag duties rotate from developer to developer on a weekly basis.
+
+Bugsnag is an important part of our developer role here. It allows us time to catch and resolve bugs and issues which we would not otherwise see or prioritise. This creates a better development environment for us all as well as improving the customers experience on their site. 
```
experience on OUR site

pmcnr-hx commented on bugsnag-process.md:
```diff
@@ -0,0 +1,149 @@
+
+The errors are split by project and you can switch between these using the menu in the top left corner.
+
+![](https://cldup.com/BjzztadbNV-1200x1200.png)
```
I recommend attaching the screenshots to this PR and linking from there, this way all the content will be hosted in GitHub without external dependencies.

Commit:
our site

Commit:
adding cleaning up section

Commit:
moving to locally hosted images

Commit:
adding missed images

Commit:
fixing filename

Commit:
adding -process

lukehansell commented:
okay, so I've added the bit about removing bugsnags (cleaning up) and I've moved the images to a local /images/bugsnag-process/ directory...

c.c. @pmcnr-hx @LouisPryer @stuartstobie @Joezo

pmcnr-hx commented:
:+1:
